.travis.yml

```diff
@@ -17,4 +17,4 @@ before_install:
   - sudo apt-get install automake autoconf intltool xsltproc libglib2.0-dev libpolkit-gobject-1-dev libpolkit-agent-1-dev phantomjs gtk-doc-tools libjson-glib-dev libnm-glib-dev libpam0g-dev libssh-dev libsystemd-daemon-dev libsystemd-journal-dev libjson-perl libjavascript-minifier-xs-perl pkg-config glib-networking valgrind libkeyutils-dev xmlto xsltproc libpcp3-dev libpcp-pmda3-dev libpcp-import1-dev pcp liblvm2-dev libgudev-1.0-dev libgirepository1.0-dev
 
 script:
-  - ./autogen.sh --prefix=/usr --enable-strict && make V=1 all && sudo make enable-root-tests && make distcheck && make -j8 check-memory
+  - gcc ./tools/careful-cat.c -o cc; set -o pipefail; (./autogen.sh --prefix=/usr --enable-strict && make V=1 all && sudo make enable-root-tests && make distcheck && make -j8 check-memory) 2>&1 | ./cc
```

Comment:
cc is a compiler on a lot of operating systems ... unfortunately.

Also do yo uthink we should split up the script into multiple lines? Does that work?